packages/xo-server-auth-ldap/src/index.js

```diff
@@ -112,6 +118,69 @@ Or something like this if you also want to filter by group:
       type: 'string',
       default: DEFAULTS.filter,
     },
+    users: {
+      title: 'Users',
+      description: 'Configure how LDAP users are added to XO',
+      type: 'object',
+      properties: {
+        idAttribute: {
+          title: 'ID attribute',
+          description:
+            'Attribute used to identify a user. Must be unique. e.g.: `dn`',
+          type: 'string',
+          default: DEFAULTS.users.idAttribute,
```

Comment:
These default values are only used for information, and not automatically merged with the config, you need to take care of them yourself.

Reply:
That's the behaviour I'm expecting in the code. e.g.: if the `idAttribute` isn't configured, I'm falling back to the old implementation.

Reply:
Then, IMHO, it's not a default value and should be removed.